server/src/main/java/org/opensearch/index/mapper/StarTreeMapper.java

```diff
@@ -463,7 +463,7 @@ protected void parseCreateField(ParseContext context) {
      * @opensearch.experimental
      */
     @ExperimentalApi
-    public static final class StarTreeFieldType extends CompositeDataCubeFieldType {
```

Comment:
Is this intentional , any reasons to remove final ?

Reply:
It was creating an issue while mocking. Making some changes now - reverting this change.